src/components/HomePage/ListOfHouses/ListOfHouses.tsx

```diff
@@ -21,6 +24,12 @@ function ListOfHouses({ houses }: { houses: BasicHouseData[] }) {
     setSortType(null);
   };
 
+  const slideProperties = {
```

Comment:
Could the speed of changing slides possibly be increased slightly?

Reply:
I changed from 1000 to 700